numpy/lib/format.py

```diff
@@ -552,7 +571,7 @@ def write_array(fp, array, version=None, allow_pickle=True, pickle_kwargs=None):
     """
     _check_version(version)
     used_ver = _write_array_header(fp, header_data_from_array_1_0(array),
-                                   version)
+                                   version)[0]
```

Comment:
It's so easy to miss the `[0]` at the end of lines like this, I prefer the style using tuple-unpacking like
```
used_ver, _ = _write_array_header(...)
```